cibuildwheel/linux.py

```diff
@@ -60,30 +60,28 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
             {environment_exports}
 
             for PYBIN in {pybin_paths}; do
-                # Setup
-                rm -rf /tmp/built_wheel
-                rm -rf /tmp/delocated_wheels
-                mkdir /tmp/built_wheel
-                mkdir /tmp/delocated_wheels
-
                 if [ ! -z {before_build} ]; then
                     PATH="$PYBIN:$PATH" sh -c {before_build}
                 fi
 
-                # Build that wheel
+                # Build the wheel
+                rm -rf /tmp/built_wheel
+                mkdir /tmp/built_wheel
                 PATH="$PYBIN:$PATH" "$PYBIN/pip" wheel . -w /tmp/built_wheel --no-deps {build_verbosity_flag}
                 built_wheel=(/tmp/built_wheel/*.whl)
 
-                # Delocate the wheel
+                # repair the wheel
+                rm -rf /tmp/repaired_wheels
+                mkdir /tmp/repaired_wheels
                 # NOTE: 'built_wheel' here is a bash array of glob matches; "$built_wheel" returns
                 # the first element
-                if [[ "$built_wheel" == *none-any.whl ]]; then
-                    # pure python wheel - just copy
-                    mv "$built_wheel" /tmp/delocated_wheels
+                if [[ "$built_wheel" == *none-any.whl ]] || [ -z {repair_command} ]; then
+                    # pure Python wheel or empty repair command
+                    mv "$built_wheel" /tmp/repaired_wheels
                 else
-                    auditwheel repair "$built_wheel" -w /tmp/delocated_wheels
+                    built_wheel=$built_wheel sh -c {repair_command}
```

Comment:
Good idea! I had to use `sh -c {repair_command} repair_command "$built_wheel"` because the format is `sh -c command_string [command_name [argument ...]]`